### PR TITLE
Update Installer.php

### PR DESCRIPTION
**Pull request by eshiol**
Pull Request for Issue # .

### Summary of Changes
Fixes a bug in the class Joomla\CMS\Installer\Installer

### CODE REVIEW

File: libraries/src/Installer/Installer.php [hunk 1]
```diff
@@ -2377,19 +2377,19 @@ public function getAdapters($options = array(), array $custom = array())
 		// Add any custom adapters if specified
 		if (count($custom) >= 1)
 		{
-			foreach ($custom as $adapter)
+			foreach ($custom as $name)
```
Comment (zero-24) [inline libraries/src/Installer/Installer.php]: hmm can we please keep the variable name `$customAdapter` and replace $name with this?

**Review by zero-24 — CHANGES_REQUESTED** [review]
As noted inline

**Review by wilsonge — APPROVED** [review]

**Comment by infograf768** [comment]
May we know which bug this is supposed to solve? And how to reproduce that bug?

**Comment by ghost** [comment]
closing due the lack of response.

**Pull request closed without merging**